app/notification.go

```diff
@@ -348,7 +348,7 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 				)
 			} else {
 				// register that a notification was not sent
-				a.NotificationsLog().Warn("Notification not sent",
+				a.NotificationsLog().Info("Notification not sent",
```

Comment:
We can go with `DEBUG` level. This must escaped from me as this is totally expected and happens for an individual user. This might be a perfect example for the `DEBUG` level. Same for the other occurrence as well.

Reply:
Changed.